app/Party.php

```diff
@@ -394,7 +401,15 @@ public function scopeUpcomingEvents($query, $by_event = false)
 
       return $this->join('groups', 'groups.idgroups', '=', 'events.group')
             ->join('users_groups', 'users_groups.group', '=', 'groups.idgroups')
-            ->whereNotNull('events.wordpress_post_id')
+            ->leftJoin('events_users', function($join) {
+                $join->on('events_users.event', '=', 'events.idevents');
+                $join->where('events_users.user', '=', Auth::User()->id);
```

Comment:
Does this run?  I thought it was Auth::user() (lowercase)

Reply:
Hmmm.  It can't be right, will change.